kfs-core/src/main/java/edu/arizona/kfs/fp/batch/service/impl/ProcurementCardHolderUpdateServiceImpl.java

```diff
@@ -0,0 +1,366 @@
+                procurementCardHolderDetail.setCardHolderAlternateName(procurementCardHolderLoad.getCardHolderAlternateName());
+                procurementCardHolderDetail.setCardHolderLine1Address(procurementCardHolderLoad.getCardHolderLine1Address());
+                procurementCardHolderDetail.setCardHolderLine2Address(procurementCardHolderLoad.getCardHolderLine2Address());
+                procurementCardHolderDetail.setCardHolderCityName(procurementCardHolderLoad.getCardHolderCityName());
+                procurementCardHolderDetail.setCardHolderStateCode(procurementCardHolderLoad.getCardHolderStateCode());
+                procurementCardHolderDetail.setCardHolderZipCode(procurementCardHolderLoad.getCardHolderZipCode());
+                procurementCardHolderDetail.setCardHolderWorkPhoneNumber(procurementCardHolderLoad.getCardHolderWorkPhoneNumber());
+                procurementCardHolderDetail.setCardLimit(procurementCardHolderLoad.getCardLimit());
+                procurementCardHolderDetail.setCardCycleAmountLimit(procurementCardHolderLoad.getCardCycleAmountLimit());
+                procurementCardHolderDetail.setCardCycleVolLimit(procurementCardHolderLoad.getCardCycleVolumeLimit());
+                procurementCardHolderDetail.setCardMonthlyNumber(procurementCardHolderLoad.getCardMonthlyNumber());
+                procurementCardHolderDetail.setCardStatusCode(determineStatusCode(procurementCardHolderLoad.getCardStatusCode()));
+                procurementCardHolderDetail.setCardCancelCode(determineCancelCode(procurementCardHolderLoad.getCardStatusCode()));
+                procurementCardHolderDetail.setCardNoteText(procurementCardHolderLoad.getCardNoteText());
+                procurementCardHolderDetail.setCardOpenDate(procurementCardHolderLoad.getCardOpenDate());
+                procurementCardHolderDetail.setCardCancelDate(procurementCardHolderLoad.getCardCancelDate());
+                if (ObjectUtils.isNotNull(procurementCardHolderLoad.getCardExpireDate())) {
+                    procurementCardHolderDetail.setCardExpireDate(determineExpireDate(procurementCardHolderLoad.getCardExpireDate()));
+                }
+                procurementCardHolderDetails.add(procurementCardHolderDetail);
+            }
+            else {
+                excludedRecords = excludedRecords + 1;
+            }
+        }
+        
+        try {
+            for(ProcurementCardDefault procurementCardDefault : procurementCardHolderDetails) {
+                if(ObjectUtils.isNull(procurementCardDefault.getId())) {
+                    procurementCardDefault.setId(Long.valueOf(new NextProcurementCardDefaultIdFinder().getValue()));
+                }
+            }
+            
+            businessObjectService.save(procurementCardHolderDetails);
+            if ( LOG.isInfoEnabled() ) {
+                LOG.info("Total Procurement Cardholders inserted: " + Integer.toString(insertedRecords));
```

Comment:
Please remove all further Integer.toString conversions in the LOG statements. Logger can figure out on its own how to print the objects